article/app/services/dotcomponents/ArticlePicker.scala

```diff
@@ -126,13 +126,12 @@ object ArticlePicker {
 
     val whitelistFeatures = featureWhitelist(page, request)
     val isEnabled = conf.switches.Switches.DotcomRendering.isSwitchedOn
-    val isAdFree = ArticlePageChecks.isAdFree(page, request)
     val isCommercialBetaUser = ActiveExperiments.isParticipating(DotcomRenderingAdvertisements)
 
     // add free pages always go through DCR provided it's turned on and we support its article features.
```

Comment:
If this gets merged, will this comment still be true?

Reply:
I m doing a refactoring that will take care of that problem: https://github.com/guardian/frontend/pull/22008